index.html

```diff
@@ -26,16 +29,34 @@ <h1 class="hero__block-heading">There is no other platforms for you as like</h1>
           <div class="design__left-box">
               <h4 class="design__sub-heading sub-heading">NEW DESIGN</h4>
               <h2 class="design__heading heading">There is no other platforms for you as like<span>....</span></h2>
-              <p class="design__description description child">Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id elit non mi porta gravida at eget metus. Curabitur blandit tempus porttitor. Etiam porta sem malesuada magna mollis euismod. Vestibulum id ligula porta felis euismod semper.</p>
+              <p class="design__description description">Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id elit non mi porta gravida at eget metus. Curabitur blandit tempus porttitor. Etiam porta sem malesuada magna mollis euismod. Vestibulum id ligula porta felis euismod semper.</p>
               <p class="design__description description"> Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Donec ullamcorper nulla non metus auctor fringilla. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit libero, a pharetra augue.</p>
           </div>
           <div class="design__right-box">
               <img class="design__delta" src="build/img/delta.png" alt="delta">
           </div>
       </section>
 
-      <section>
-
+      <section class="awesone-features">
+          <h4 class="awesone-features__sub-heading sub-heading">NEW FEATURES</h4>
+          <h2 class="awesone-features__heading heading">Some awesone features</h2>
+          <div class="awesone-features__column">
+              <div class="awesone-features__card">
+                  <img class="awesone-features__cardbg" src="build/img/photos.png" alt="photos">
+                  <h5 class="awesone-features__card-title">Some awesone features</h5>
+                  <p class="description">Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit libero, a pharetra augue.</p>
```

Comment:
Rename to awesone-features__description

Reply:
This class is named "description" bacause he add to global.scss.